clang/test/Driver/cl-x86-flags.c

```diff
@@ -69,7 +69,10 @@
 // RUN: %clang_cl -m32 -arch:avx2 --target=i386-pc-windows -### -- 2>&1 %s | FileCheck -check-prefix=avx2 %s
 // avx2: invalid /arch: argument
 
-// RUN: %clang_cl -m32 -arch:AVX512F --target=i386-pc-windows /c /Fo%t.obj -Xclang -verify -DTEST_32_ARCH_AVX512F -- %s
+// RUN: %clang_cl -m32 -arch:AVX512F --target=i386-pc-windows /c /Fo%t.obj -Xclang -verify=KNL1 -DTEST_32_ARCH_AVX512F -- %s
```

Comment:
Why we emit warnings given no KNL/KNM feature specified?

Reply:
AVX512F will be handled as "knl" in driver

Reply:
Do you mean AVX512PF? By clang-cl?

Reply:
https://github.com/llvm/llvm-project/blob/87d15d7ad6141d7492e9579928839408d60e8152/clang/lib/Driver/ToolChains/Arch/X86.cpp#L45